templates/email/default/archive-old-enquiries.txt

```diff
@@ -0,0 +1,35 @@
+Subject: Your reports on [% site_name %]
+
+Hello [% user.name %],
+
+FixMyStreet is being updated in [% council_name %] to improve how problems get reported.
+
+As part of this process we are closing all reports made before the update.
+
+We noticed that you have [% report_count %] old [% nget('report', 'reports', report_count) %] on the system, which we've listed below.
+
+All of your reports will have been received and reviewed by [% council_name %], so if your report is no longer an issue, you don't need to do anything.
+
+[% IF council_url %]
```

Comment:
This is a text email, not HTML. Indenting IF statements will appear indented in the email :)